buildpipeline/perf-pipeline.groovy

```diff
@@ -0,0 +1,390 @@
+@Library('dotnet-ci') _
+
+// Incoming parameters.  Access with "params.<param name>".
+// Note that the parameters will be set as env variables so we cannot use names that conflict
+// with the engineering system parameter names.
+
+//--------------------- Windows Functions ----------------------------//
+
+def windowsBuild(String arch, String config, String pgo, boolean isBaseline) {
+    checkout scm
+
+    String pgoStash = ((pgo == "skiprestoreoptdata") ? "nopgo" : "pgo")
```

Comment:
nit: to better parallel what we do in the tests (versus the builds): I think that the variable `${pgo}` should always be either `'pgo'` or `'nopgo'`; perhaps the build flag (`''` or `'nopgooptimize'`) could be stored in a variable named `pgoBuildFlag` or something similar.

Reply:
Sure thing